app/controllers/concerns/medicaid_flow.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module MedicaidFlow
+  def ensure_application_present
+    if current_medicaid_application.blank?
+      redirect_to medicaid_intro_location_steps_path
+    end
+  end
+
+  def current_medicaid_application
+    MedicaidApplication.find_by(id: current_medicaid_application_id)
+  end
+
+  def current_medicaid_application_id
+    session[:medicaid_application_id]
+  end
+end
```

Comment:
If we change the references to `current_medicaid_application` to `current_application` then the two different flows (both `MedicaidFlow` and `SnapFlow`) always have the same method as the entry point to that object. The two of them just access the records in their own way - same interface, different objects returned :)

Reply:
yes I want to do this in the next PR to separate that change from these changes -- that work for you?

Reply:
AH! Yeah!